src/Tabs/Tabs.js

```diff
@@ -207,7 +207,7 @@ class Tabs extends Component {
       return React.cloneElement(tab, {
         key: index,
         selected: this.getSelected(tab, index),
-        tabIndex: index,
+        tabIndex: 0,
```

Comment:
Good point. `tabIndex: 0` and omitting the tab index have the same behavior. Also, this allows you to specify `tabIndex` on the `Tab` component.